### ISSUE TITLE
Issues with Reproducing Experiment Results

### ISSUE THREAD

**Issue by VG-Huai** [rccchoudhury/rlt#11]
Thank you so much for open-sourcing this work! 
Now I’ve been working on a redesigned efficient tokenization method based on your repository, which requires retraining and testing. However, I’ve run into some issues while attempting to reproduce your experiment results from scratch.
Specifically, I trained a baseline model without skipping any tokens using the following configuration:
```
# @package _global_

# to execute this experiment run:
# python train.py experiment=val_vit_mae.yaml

defaults:
  - override /data: kinetics
  - override /model: vit_mae_oct_256 # change to 256 resolution
  - override /callbacks: default
  - override /trainer: default
  #- override /logger: wandb

# all parameters below will be merged with parameters from default configurations set above
# this allows you to overwrite only specified parameters

tags: ["kinetics400"]

seed: 42

task_name: "vitb_rlt_without_depth"

train: True

trainer:
  precision: 16
  devices: 1
  accelerator: "gpu"
  min_epochs: 0
  max_epochs: 100
  default_root_dir: ${paths.output_dir}
  plugins: 
    _target_: pytorch_lightning.plugins.environments.LightningEnvironment


model:
  compile: true
  pretrain: /home/ehliu/projects/oct/checkpoints/vit_g_hybrid_pt_1200e.pth
  model:
    use_flash_attn: true
    #use_length_embed: true # fix true
    use_length_embed: false
  scheduler_cfg: 
    _target_: src.models.optim_utils.CosineSchedulerConfig
    # Partial bc batch size depends on other stuff.
    warmup_epochs: 5
    total_epochs: 100
    lr: 1.5e-3
    end_lr: 1e-6
    start_lr: 1e-6
  optimizer: 
    lr: 1
    # lr: 1.5e-3
    _target_: torch.optim.AdamW
    _partial_: true
    betas: [0.9, 0.999]
    weight_decay: 0.05
  tokenizer_cfg:
    _target_: src.models.tokenizer_utils.TokenizerConfig
    drop_policy: None
    drop_param: 0.1
    encode_length: False
    mixup_config:
      _target_: src.models.tokenizer_utils.MixupConfig
      mixup_alpha: 0.0
      cutmix_alpha: 1.0
      cutmix_minmax: None
      prob: 1.0
      # prob: 0.5
      switch_prob: 0.5
      mode: batch
      label_smoothing: 0.1
      num_classes: 400
    re_config:
      _target_: src.models.tokenizer_utils.RandomErasingConfig
      probability: 0.25
      mode: const
      device: cuda
      min_count: 1
    # rand_aug_config:
    #   _target_: src.models.tokenizer_utils.RandAugmentConfig
    #   num_layers: 4
    #   magnitude: 7

  
data:
  batch_size: 24
  num_workers: 24
  
# logger: 
#   wandb:
#     _target_: pytorch_lightning.loggers.wandb.WandbLogger
#     name: ${task_name}
#     project: "pregrouping"
#     tags: ${task_name}
#     group: "baselines"
```
I trained the model on the Kinetics400 dataset with these modifications:

- Batch size: 24
- Resolution: 256 (changed from 224)
- Pre-trained model: vit_g_hybrid_pt_1200e.pth from the [VideoMAEv2 Model Zoo](https://github.com/OpenGVLab/VideoMAEv2/blob/master/docs/MODEL_ZOO.md)

After training, I only achieved an accuracy of 58.98% on val set, which seems quite low. I’m wondering if I might have used the wrong pre-trained model. Could you please let me know if vit_g_hybrid_pt_1200e.pth is the correct choice for this experiment? Thank you so much for your time and help!

![Image](https://github.com/user-attachments/assets/383b479e-7524-4f98-93e5-f2c04350b738)
